### ISSUE TITLE
Laggy popups

### ISSUE THREAD

**Issue by Saroufim** [tkashkin/eOSSteamSkin#10]
Hovering over items creates popups that are rather slow to render. Would it be possible to disable them?

**Comment by tkashkin** [comment]
Probably no. Steam skins are quite limited. It's only possible to change control appearance and position. It's impossible to add or remove controls (except maybe labels and links). It's even impossible to properly hide something.

Are you using this skin on Linux? For some reason it lags way more on Linux than on other systems.

It may be possible to optimize this skin, but I don't really know what did I do wrong and why does it lag so much.

**Comment by Saroufim** [comment]
Yes I'm using it on Linux with Flatpak specifically. It's a much needed skin to make steam look native to be honest. Thanks for the effort.

**Comment by tkashkin** [comment]
Maybe there are some skin devs who know what can cause such performance drop and how to optimize it. I don't know and haven't seen much info about it.
Maybe you know some skins which had same issues previously but have somehow fixed it?
